packages/wonder-stuff-ci/src/exec-prom.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * A simple promisified version of child_process.exec, so we can `await` it
+ */
+import {exec, ExecOptions} from "child_process";
+import util from "util";
+
+export const bufferToString = (input: Buffer | string): string => {
+    return typeof input === "string" ? input : input.toString("utf8");
+};
+
+export const execProm = util.promisify(exec);
```

Comment:
**suggestion:** Generally, when promisifying async methods, I have used the convention of adding `Async` to the name. It is generally clearer to grasp and understand what the function is for in that scenario.

That said, I'm not sure it's worth the abstraction of its own file rather than just creating the method in the files that use it - except maybe to simplify testing?

When you change the name of the function, don't forget to change the name of the file to match.

```suggestion
export const execAsync = util.promisify(exec);
```